### PR TITLE
bugreportz, cmd: add Nepali translation

### PR DESCRIPTION
**Pull request by subaseyy**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ne/android/bugreportz.md
Comment (Magrid0) [inline pages.ne/android/bugreportz.md:9]:
```suggestion
`bugreportz`
```

**Review by Magrid0 — CHANGES_REQUESTED** [review]
just this small change and your PR should be ready to merge, after it get approved from another maintainer and the codeowners

File: pages.ne/android/cmd.md
Comment (kbdharun) [inline pages.ne/android/cmd.md:16]:
```suggestion
`cmd {{सेवा}}`

- विशिष्ट तर्क संग सेवा कल गर्नुहोस्:

`cmd {{सेवा}} {{तर्क1 तर्क2 ...}}`
```

**Review by kbdharun — APPROVED** [review]

File: pages.ne/android/bugreportz.md
Comment (Magrid0) [inline pages.ne/android/bugreportz.md:5]:
```
> थप जानकारी: <https://cs.android.com/android/platform/superproject/+/main:frameworks/native/cmds/bugreportz>.
```

Shouldn't it be like this?

Comment (kbdharun) [reply pages.ne/android/bugreportz.md:5]: Nice catch, the page descriptions use the native period whereas more info link one used the conventional period.

File: pages.ne/android/cmd.md
Comment (kbdharun) [inline pages.ne/android/cmd.md:4]:
```suggestion
> थप जानकारी: <https://cs.android.com/android/platform/superproject/+/main:frameworks/native/cmds/cmd/>.
```

File: pages.ne/android/bugreportz.md
Comment (kbdharun) [reply pages.ne/android/bugreportz.md:5]:
```suggestion
> थप जानकारी: <https://cs.android.com/android/platform/superproject/+/main:frameworks/native/cmds/bugreportz>.
```

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

**Pull request merged**
